modules/nw-osp-enabling-ovs-offload.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-enabling-ovs-offload_{context}"]
+= Enabling OVS hardware offloading
+
+For clusters that run on {rh-openstack-first}, you can enable link:https://www.openvswitch.org/[Open vSwitch (OVS)] hardware offloading. 
+
+OVS is a multi-layer virtual switch that enables large-scale, multi-server network virtualization. 
+
+.Prerequisites
+
+* You installed a cluster on {rh-openstack} that is configured for single-root input/output virtualization (SR-IOV).
+* You installed the SR-IOV network Operator on your cluster. 
+* You created two `hw-offload` type virtual function (VF) interfaces on your cluster.
+
+.Procedure
+
+. From a command line, disable the admission webhook:
++
+[source,terminal]
+----
+$oc patch sriovoperatorconfig default --type=merge -n openshift-sriov-network-operator --patch '{ "spec": { "enableOperatorWebhook": false } }'
+----
+
+3. Create an `SriovNetworkNodePolicy` policy for two hw-offload vf interfaces that has been pre-created in the OCP.
++
+.The first virtual function interface
+[source,yaml]
+---
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: "hwoffload9"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: true
+  nicSelector:
+    pfNames:
+    - ens6
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: 'true'
+  numVfs: 1
+  priority: 99
+  resourceName: "hwoffload9"
+----
++
+.The second virtual function interface
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy <1>
+metadata:
+  name: "hwoffload10"
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  isRdma: true
+  nicSelector:
+    pfNames: <2>
+    - ens5
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: 'true'
+  numVfs: 1
+  priority: 99
+  resourceName: "hwoffload10"
+----
+<1> Insert the `SriovNetworkNodePolicy` value here.
+<2> Both interfaces must include physical function (PF) names.
+
+4. Create NetworkAttachmentDefinition for two interfaces that has been created in the above step #3
```

Comment:
Thanks! I also just pushed a few changes--might want to re-check the latter steps now.

Reply:
actually nevermind, we need it:
configure a network-attachment definition using host-device - this is needed because we must past the kernel nic into the host to support the dpdk (https://doc.dpdk.org/guides/howto/flow_bifurcation.html) and we can't use the sriov-cni as there is no PF on the host.
Maybe we can add a note about it.

Source: https://bugzilla.redhat.com/show_bug.cgi?id=2036948#c4